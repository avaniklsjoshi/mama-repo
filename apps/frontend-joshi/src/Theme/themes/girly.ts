export default {
  themeType: "girly",
  colors: {
    background: "#f1b7ea",
    text: "#fff"
  },
  header: {
    background: "#000",
    text: "#fff"
  },
  footer: {
    background: "#262726",
    text: "#fff"
  },
  section: {
    background: "#008F7A",
    text: "#fff"
  }
};
