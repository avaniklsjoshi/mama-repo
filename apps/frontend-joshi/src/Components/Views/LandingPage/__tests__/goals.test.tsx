/* eslint-disable testing-library/no-node-access */
/* eslint-disable testing-library/no-container */
import { act, fireEvent, render, screen } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import girly from "../../../../Theme/themes/girly";
import Goals from "../goals";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key: any) => key })
}));

describe("Landing Page", () => {
  it("renders Goals", () => {
    const { container } = render(
      <ThemeProvider theme={girly}>
        <Goals />
      </ThemeProvider>
    );
    expect(container).toMatchSnapshot();
    expect(screen.getByText("myInterests")).toBeInTheDocument();
  });
});
