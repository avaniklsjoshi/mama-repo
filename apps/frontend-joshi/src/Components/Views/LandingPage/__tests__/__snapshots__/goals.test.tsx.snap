// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page renders Goals 1`] = `
<div>
  <div
    class="goals-container"
  >
    <div
      class="sc-bczRLJ jliZnU goals-title"
      data-testid="goals-title"
    >
      myInterests
    </div>
    <ul />
  </div>
</div>
`;
